x-pack/plugins/apm/public/context/UrlParamsContext/helpers.ts

```diff
@@ -80,6 +80,12 @@ export function getPathParams(pathname: string = '') {
         processorEvent: 'metric',
         serviceName: paths[0]
       };
+    case 'services': // fall thru since services and traces share path params
+    case 'traces':
+      return {
+        processorEvent: 'transaction',
+        serviceName: undefined
+      };
```

Comment:
This is okay for now. Going forward we should really get rid of this and instead use the routes in [routeConfig](https://github.com/elastic/kibana/blob/9d759a273e6d3b3aa6a5396c89b5176883e1ba14/x-pack/plugins/apm/public/components/app/Main/routeConfig.tsx) to get the path params for the current location.

We can use something like [`path-to-regexp`](https://github.com/pillarjs/path-to-regexp) which is used by react-router under the hood.